src/index.d.ts

```diff
@@ -50,7 +50,7 @@ export class Evaluator {
 export class Operation extends Brush {
 
   isOperation: boolean;
-  // operation
+  operation: number;
```

Comment:
Thank you! Lets change this to `CSGOperation`. And while we're here can you add the new `HOLLOW_SUBTRACTION` and `HOLLOW_INTERSECTION` operations to the CSGOperation enum above?

Reply:
Sorry, totally missed this. Updated :)